src/TwigExtensionFactory.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive-twigrenderer for the canonical source repository
+ * @copyright Copyright (c) 2018 Zend Technologies USA Inc. (https://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive-twigrenderer/blob/master/LICENSE.md New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Zend\Expressive\Twig;
+
+use ArrayObject;
+use Psr\Container\ContainerInterface;
+use Zend\Expressive\Helper\ServerUrlHelper;
+use Zend\Expressive\Helper\UrlHelper;
+use Zend\Expressive\Twig\Exception\InvalidConfigException;
+
+class TwigExtensionFactory
+{
+    public function __invoke(ContainerInterface $container) : TwigExtension
+    {
+        if (! $container->has(ServerUrlHelper::class)) {
+            throw new InvalidConfigException(sprintf(
+                'Missing required `%s` dependency.',
```

Comment:
I would add more detailed exception message, see for example:
https://github.com/zendframework/zend-expressive-router/blob/develop/src/Exception/MissingDependencyException.php#L21-L27